scripts/sct_denoising_onlm.py

```diff
@@ -97,78 +159,32 @@ def main(file_to_denoise, param, output_file_name) :
     # Save files
     img_denoise = nib.Nifti1Image(den, None, hdr_0)
     img_diff = nib.Nifti1Image(diff_3d, None, hdr_0)
-    if output_file_name != None :
+    if output_file_name != None:
```

Comment:
i think in Python the "is not None" is preferred than "!= None"

Reply:
Ok sounds good. Changed in commit [8ccc505](https://github.com/neuropoly/spinalcordtoolbox/commit/8ccc505b6ce522a0f6f7e66b99ed33b1ddb067c3)